src/routes/about/+page.svelte

```diff
@@ -20,4 +25,11 @@
             {@html data.partial}
         </div>
     </div>
+    {#if bottomNavBarEnabled}
+        <div class="bottom-nav-bar">
```

Comment:
What are the divs for?

I made it this and it worked fine:

```
    {#if bottomNavBarEnabled}
        <BottomNavigationBar {barType} />
    {/if}
```

Reply:
Removed